patchwork-events-input/src/main/java/net/patchworkmc/mixin/event/input/MixinKeyboard.java

```diff
@@ -33,11 +34,12 @@
 @Mixin(Keyboard.class)
 public abstract class MixinKeyboard {
 	@Shadow
-	MinecraftClient client;
+	@Final
+	private MinecraftClient client;
 
 	@Inject(method = "onKey", at = @At("RETURN"))
```

Comment:
Use an ordinal, change to TAIL, or comment you're specifically targeting all return points.